cwms_radar_api/src/main/java/cwms/radar/data/dto/rating/RatingSpec.java

```diff
@@ -0,0 +1,473 @@
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonNaming(PropertyNamingStrategies.KebabCaseStrategy.class)
+public class RatingSpec implements CwmsDTO
+{
+	private final String officeId;
+	private final String ratingId;
+	private final String templateId;
+	private final String locationId;
+	private final String version;
+	private final String sourceAgency;
+
+	private final String inRangeMethod;
+	private final String outRangeLowMethod;
+	private final String outRangeHighMethod;
+	private final boolean active;
+	private final boolean autoUpdate;
+	private final boolean autoActivate;
+	private final boolean autoMigrateExtension;
+	private final IndRoundingSpec[] indRoundingSpecs;
+	private final String depRoundingSpec;
+	private final String description;
+
+	@JsonFormat(shape = JsonFormat.Shape.STRING)
+	private List<ZonedDateTime> effectiveDates;
+
+
+
+	public RatingSpec(Builder builder){
+		this.officeId = builder.officeId;
+		this.ratingId = builder.ratingId;
+		this.templateId = builder.templateId;
+		this.locationId = builder.locationId;
+		this.version = builder.version;
+		this.sourceAgency = builder.sourceAgency;
+		this.inRangeMethod = builder.inRangeMethod;
+		this.outRangeLowMethod = builder.outRangeLowMethod;
+		this.outRangeHighMethod = builder.outRangeHighMethod;
+		this.active = builder.active;
+		this.autoUpdate = builder.autoUpdate;
+		this.autoActivate = builder.autoActivate;
+		this.autoMigrateExtension = builder.autoMigrateExtension;
+		this.indRoundingSpecs = builder.indRoundingSpecs;
+		this.depRoundingSpec = builder.depRoundingSpec;
+		this.description = builder.description;
+		this.effectiveDates = builder.effectiveDates;
+	}
+
+	public String getOfficeId()
+	{
+		return officeId;
+	}
+
+	public String getRatingId()
+	{
+		return ratingId;
+	}
+
+	public String getTemplateId()
+	{
+		return templateId;
+	}
+
+	public String getLocationId()
+	{
+		return locationId;
+	}
+
+	public String getVersion()
+	{
+		return version;
+	}
+
+	public String getSourceAgency()
+	{
+		return sourceAgency;
+	}
+
+	public String getInRangeMethod()
+	{
+		return inRangeMethod;
+	}
+
+	public String getOutRangeLowMethod()
+	{
+		return outRangeLowMethod;
+	}
+
+	public String getOutRangeHighMethod()
+	{
+		return outRangeHighMethod;
+	}
+
+	public boolean isActive()
+	{
+		return active;
+	}
+
+	public boolean isAutoUpdate()
+	{
+		return autoUpdate;
+	}
+
+	public boolean isAutoActivate()
+	{
+		return autoActivate;
+	}
+
+	public boolean isAutoMigrateExtension()
+	{
+		return autoMigrateExtension;
+	}
+
+	public IndRoundingSpec[] getIndRoundingSpecs()
+	{
+		return indRoundingSpecs;
+	}
+
+	public String getDepRoundingSpec()
+	{
+		return depRoundingSpec;
+	}
+
+	public String getDescription()
+	{
+		return description;
+	}
+
+	public List<ZonedDateTime> getEffectiveDates() {
+		return effectiveDates;
+	}
+
+	@Override
+	public void validate() throws FieldException
+	{
+
+	}
+
+	public static class IndRoundingSpec {
```

Comment:
I feel if the full name is spelled out in https://github.com/USACE/cwms-radar-api/pull/164/files#diff-5f04b8de36b6b1f611173776126c3d052b2d15f689420e6339ff0b9a82ec7e46 then it probably should be here as well, just for sake of consistency. So `IndependentRoundingSpec`, for example.

Same for the properties in parent class (e.g. `getIndependentRoundingSpecs()`, `getDependentRoundingSpecs()`).

Reply:
Agreed